gcloud/test__helpers.py

```diff
@@ -221,6 +222,15 @@ def test_read_from_cli_info(self):
         project_id = self.callFUT('test-project-id')
         self.assertEqual('test-project-id', project_id)
 
+    def test_gae_without_expanduser(self):
+        import sys
+        try:
+            sys.modules['pwd'] = None  # Blocks pwd from being imported.
```

Comment:
Other tests _should_ attempt to import it if they need it. However if it `sys.modules['pwd'] = None` then it won't try and import it again. So that's why I have `del sys.modules['pwd']` at the end there.